src/Utils/types.ts

```diff
@@ -0,0 +1,10 @@
+export type Login = {
+    username: string;
+    password: string;
+};
+
+export type MappedResponse<T> = {
+    data: boolean | T | undefined;
```

Comment:
`data?: boolean | T`

Instead of **| undefined**

Reply:
Done